ES/ESum042.xml

```diff
@@ -644,250 +355,425 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                 <locus target="#105v"/>: <foreign xml:lang="gez">በልደቱ፡ ለመድኃኒነ</foreign>,
+                                 <locus target="#108v"/>: <foreign xml:lang="gez">በአስተርእዮ</foreign>,
+                                 <locus target="#112v"/>: <foreign xml:lang="gez">በሞታ፡ በፍልሰታ</foreign>,
+                                 <locus target="#113v"/>: <foreign xml:lang="gez">በፍልሰታ፡</foreign>, 
+                                 <locus target="#115r"/>: <foreign xml:lang="gez">በሕንጸታ፡ በቅዳሴ</foreign>,  
+                                 <locus target="#117r"/>: <foreign xml:lang="gez">በሕንጸታ።</foreign>,  
+                                 <locus target="#118v"/>: <foreign xml:lang="gez">በመስቀል</foreign>,  
+                                 <locus target="#124v"/>: <foreign xml:lang="gez">በሚካኤል</foreign>.
+                                 </desc>                             
                         </item>
                      </list>
                   </additions>
-                  <bindingDesc>
-                     <binding contemporary="false">
-                        <decoNote xml:id="b1">Two wooden boards (the front board is broken).&#xD;
-                           Two pairs of sewing stations.</decoNote>
-                        <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                        <decoNote xml:id="b3" type="Endbands"/>
-                        <decoNote xml:id="b4" type="Other"/>
-                        <decoNote xml:id="b5">None.</decoNote>
-                     </binding>
-                  </bindingDesc>
-               </physDesc>
-               <history>
-                  <origin>
-                     <origPlace>
-                        <placeName ref="INS0017UM">ʿUra Qirqos / UQ</placeName>
-                     </origPlace>
-                     <origDate notBefore="1750" notAfter="1800">Second half of the 18th cent.
-                        (?)</origDate>
-                  </origin>
-               </history>
-               <additional>
-                  <adminInfo>
-                     <recordHist>
-                        <source>
-                           <listBibl type="catalogue">
-                              <bibl>
-                                 <ptr target="bm:EthioSpare"/>
-                              </bibl>
-                           </listBibl>
-                        </source>
-                     </recordHist>
-                     <custodialHist>
-                        <custEvent type="restorations" subtype="modern"/>
-                     </custodialHist>
-                  </adminInfo>
-
-
-               </additional>
+                  </physDesc>                     
+                          <history>
+<origin>
+   <origDate notBefore="1740" notAfter="1800">Around the middle of in the second half of the 18th century</origDate>.
```

Comment:
the fullstop after /origDate should be deleted or go inside of the tag

Reply:
ok